metricbeat/module/kubernetes/apiserver/_meta/fields.yml

```diff
@@ -147,7 +147,7 @@
         - name: duration.us.count
           type: long
           description: Request duration, number of operations
-        - name: duration.us.bucket.*
+        - name: duration.bucket.us.*
```

Comment:
`us` goes first in order to have the buckets in the same "object" with `duration.us.sum` and `duration.us.count`. Is this change intended?

Reply:
Yes because without this change the test I added for kubernetes was failing. I will look into why its failing, maybe the code needs to be fixed not the field.

Reply:
I was able to remove this for now, as I removed the field validation from the tests. I want to add that in a follow up PR, but this PR is already way to big.